### DISCUSSION TITLE
IOS GPX Line Thickness

### DISCUSSION THREAD

**Discussion by mdd423-git** [osmandapp/OsmAnd#12963] [Q&A]
Does anyone know how to change the GPX line thickness in IOS? It was pretty simple from the android app but now that I've switched to the dark side and went to IOS, I'm not finding anything on gpx line thickness.
Thanks.

**Comment by vshcherb** [comment]
It's coming 4.1 release to iOS

**Reply by mdd423-git** [reply]
Awesome! do you know if there will be anyway to color change multiple tracks from a single KMZ file?

**Reply by vshcherb** [reply]
I'm not sure... OsmAnd targets support GPX first, so it will probably require to split KMZ to multiple GPX files
